.markdownlint-cli2.jsonc

```diff
@@ -79,5 +79,11 @@
     "MD050": {
       "style": "asterisk"
     }
-  }
+  },
+  "customRules": [
```

Comment:
Add some test markdown files that can be checked when custom rules are added/updated